make/autoconf/spec.gmk.in

```diff
@@ -455,6 +455,9 @@ ifeq ($(ASAN_ENABLED), yes)
   endif
 endif
 
+# UndefinedBehaviorSanitizer
+UBSAN_ENABLED:=@UBSAN_ENABLED@
```

Comment:
I don't see anything reading this. ??

Reply:
To be clear there was a reason that `ASAN_ENABLED` was originally added to spec.gmk.in (though there was a suggestion it could/would be removed later), and no reason has been given for `UBSAN_ENABLED` being needed.

Reply:
Fair. I'll remove it. My previous comment, that I dropped on the floor. was asking about that basically.